plugins/ril.c

```diff
@@ -339,8 +339,8 @@ static void ril_connected(struct ril_msg *message, gpointer user_data)
 	struct ofono_modem *modem = (struct ofono_modem *) user_data;
 	struct ril_data *rd = ofono_modem_get_data(modem);
 
-	ofono_info("[%d,UNSOL]< %s", g_ril_get_slot(rd->ril),
-		g_ril_unsol_request_to_string(rd->ril, message->req));
+	g_ril_set_version(rd->ril,
+				g_ril_unsol_parse_connected(rd->ril, message));
```

Comment:
Please keep the ofono_info() message here. Adding the version to the trace would be good too.